src/lib/data/buyables/tripBuyables.ts

```diff
@@ -0,0 +1,162 @@
+import { Bank } from 'oldschooljs';
+
+import type { Buyable } from './buyables';
+
+export const tripBuyables: Buyable[] = [
```

Comment:
If these tripBuyables are entirely separate from normal buyables, which it seems like they are, they could instead not use the `Buyable` interface, or be

```ts
export const tripBuyables: ({item: EItem} & Omit<Buyable, 'name' | 'outputItems'>)[] = [
{
		item: EItem.PINEAPPLE,
		gpCost: 5,
		quantityPerHour: 3500,
		shopQuantity: 15,
		changePer: 2
	},
```

Just an idea, not required